primitives/src/signatures/bls_over_bn254.rs

```diff
@@ -120,6 +125,93 @@ impl SignatureScheme for BLSOverBN254CurveSignatureScheme {
     }
 }
 
+impl AggregateableSignatureSchemes for BLSOverBN254CurveSignatureScheme {
```

Comment:
are there any references we can add in the comments? (because even BLS signature, there are different variants of aggregation, right?)